aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -513,15 +466,14 @@ module aptos_token::token {
     }
 
     /// Transfers `amount` of tokens from `from` to `to`.
-    public entry fun transfer(
+    public fun transfer(
```

Comment:
I thought we were going to guard this with a bit in the TokenStore?

Reply:
you mean implementing have a flag for freezing tokens? Let's do it in follow-up PR